paper/paper.md

```diff
@@ -0,0 +1,112 @@
+---
+title: 'MSMetaEnhancer: A Python package for mass spectra metadata annotation'
+tags:
+  - Python
+  - mass spectra
+  - annotation
+authors:
+  - name: Matej Troják
+    orcid: 0000-0003-0841-2707
+    affiliation: 1
+  - name: Helge Hecht
+    orcid: 0000-0001-6744-996X
+    affiliation: 1
+  - name: Martin Čech
+    orcid: 0000-0002-9318-1781
+    affiliation: 1
+  - name: Elliott James Price
+    orcid: 0000-0001-5691-7000
+    affiliation: 1
+
+affiliations:
+ - name: RECETOX, Faculty of Science, Masaryk University, Kotlářská 2, Brno 60200, Czech Republic
+   index: 1
+date: 11 January 2022
+bibliography: paper.bib
+ 
+---
+
+# Summary
+
+MSMetaEnhancer is a Python software package for the metadata enrichment of entries in spectral library files. 
+The package uses matchms [@Huber2020] for data IO and supports the `.msp` open format.
+It annotates given mass spectra file by adding missing metadata such as SMILES, InChI, and CAS numbers.
+The package retrieves the medatada by querying several external databases, 
+currently supporting the chemical identifier resolver (CIR), chemical translation service (CTS) [@Wohlgemuth2010], ChemIDplus [@tomasulo2002chemidplus], and PubChem [@kim2021pubchem].
+The package is hosted via bioconda [@bioconda] and is available on Galaxy [@galaxy; @umsa].
```

Comment:
```suggestion
```
This part is now also included at the end of the software package section in more detail, so it isn't required in this place as well I think.